packages/blockchain-wallet-v4-frontend/src/components/CopyClipboardButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as C from 'services/AlertService'
+import { actions } from 'data'
+import { bindActionCreators } from 'redux'
+import { connect, ConnectedProps } from 'react-redux'
+import CopyClipboard from './template'
+import React from 'react'
+
+export interface OwnProps {
+  address: string
+  alertActions: any
```

Comment:
Can you change `address` => `text`?

If you're using ConnectedProps<typeof connector> you don't need to set alertActions here. I would consider `OwnProps` as the props that are passed in. `alertActions` is connected via mapDispatchToProps, so it's not passed in from a parent.